### DISCUSSION TITLE
Types and Imports not there

### DISCUSSION THREAD

**Discussion by livevsonline** [TanStack/router#506] [General]
Hi i'm checking out TanStack Router.

The very basic example provided in the documentation here: https://tanstack.com/router/v1/docs/quick-start

Does not currently correlate with the types in the package,

One example would be the createRouteConfig which are not exported in @tanstack->react-router->build->types->index.d.ts

```
import {
	Outlet,
	RouterProvider,
	Link,
	ReactRouter,
	createRouteConfig,
} from '@tanstack/react-router';
```

In general there is alot of typescript errors starting up a new install following the documentation above, im on version: 0.0.1-beta.61

What am i missing or doing wrong? :)

**Comment by tannerlinsley** [comment]
Fixed

**Reply by livevsonline** [reply]
Awesome, i found later on samples that worked perfectly fine, was just a little "oohh no" as it is the first sample that meets you :)
